Userland/Services/SystemServer/Service.h

```diff
@@ -71,7 +71,7 @@ class Service final : public Core::Object {
     // Whether several instances of this service can run at once.
     bool m_multi_instance { false };
     // Environment variables to pass to the service.
-    Vector<DeprecatedString> m_environment;
+    DeprecatedString m_environment;
```

Comment:
Feels like there's more than just "propagate errors" going on here. You're also changing the way environment and extra arguments are passed, but there's no mention of it in the commit message?

Reply:
Commit message updated